config/jobs/cadvisor/cadvisor.yaml

```diff
@@ -52,7 +52,6 @@ presubmits:
           - /workspace/scenarios/kubernetes_e2e.py
         args:
         - --deployment=node
-        - --gcp-project=cadvisor-e2e
```

Comment:
I think you have to add --gcp-project-type ?

Reply:
Is this e2e doing anything special? usually fixed projects were e.g. needing GPU resources or something like that, which might need a particular project pool.

Reply:
@BenTheElder i picked a random ci job to look and did not find `gcp-project-type`, let's get this in, if it fails, i can rev it again.

https://github.com/kubernetes/test-infra/blob/51cd2d4cd5972920208bf169c9c5e86d287e3b97/config/jobs/kubernetes/sig-node/containerd.yaml#L1259-L1271

Reply:
> Is this e2e doing anything special?

not as far as i know... cc @bobbypage